contrib/experimental/great_expectations_experimental/expectations/expect_queried_column_to_have_n_distinct_values_with_condition.py

```diff
@@ -70,7 +70,6 @@ def _validate(
         runtime_configuration: dict = None,
         execution_engine: ExecutionEngine = None,
     ) -> Union[ExpectationValidationResult, dict]:
-
```

Comment:
You have a whitespace change here.

Reply:
@Kilo59 Yes -- I deleted it (do we need it?).